boa_tester/src/main.rs

```diff
@@ -375,6 +397,9 @@ struct Test {
     name: Box<str>,
     description: Box<str>,
     esid: Option<Box<str>>,
+    spec_version: Option<i8>,
```

Comment:
No need to make it an `Option<i8>`, we can be conservative and assume any tests that don't have `es` ids are from the latest version (ES13). An `u8` should be enough.